### ISSUE TITLE
Energy-Systems-Components

### ISSUE THREAD

**Issue by christophwaibel** [architecture-building-systems/hive#123]
High Priority for Hive 0.2!

More Energy Systems components.

Simple Equations from Energy and Climate Systems Classes.
Check with Illias and make a list of which technologies would be important to have first.

**Comment by christophwaibel** [comment]
Energy Systems components to-do list
- [ ] PV
- [ ] Solar Thermal
- [ ] ASHP
- [ ] GSHP
- [ ] Chiller
- [ ] CHP, Micro and large
- [ ] Boiler, nat gas, wood chips, oil?
- [ ] ...?

**Comment by christophwaibel** [comment]
Duplicate https://github.com/architecture-building-systems/hive/issues/86

**Comment by christophwaibel** [comment]
Duplicate: https://github.com/architecture-building-systems/hive/issues/79

**Comment by hillias** [comment]
Focus on PV, ST, heat pump and borehole. See equations attached. For heat pump calculate COP as a function of inlet (depending on choice of anergy source, e.g. air, borehole, ...) and outlet temperature (depending on choice of heat dissipation, e.g. floor heating, radiator, ...)

[EK1_Formelsammlung_HS19_HK.pdf](https://github.com/architecture-building-systems/hive/files/4096700/EK1_Formelsammlung_HS19_HK.pdf)

**Comment by christophwaibel** [comment]
> Focus on PV, ST, heat pump and borehole. See equations attached. For heat pump calculate COP as a function of inlet (depending on choice of anergy source, e.g. air, borehole, ...) and outlet temperature (depending on choice of heat dissipation, e.g. floor heating, radiator, ...)
> 
> [EK1_Formelsammlung_HS19_HK.pdf](https://github.com/architecture-building-systems/hive/files/4096700/EK1_Formelsammlung_HS19_HK.pdf)

@hillias @amraladdin Some of those equations are not useful / incomplete for architecture students. "Systemdesign - Heizen/Kühlen": partly are over-simplified equations, that cannot be related to what architects deal with. 
E.g.:
 - "Entzugsleistung Erdsonde", what are they supposed to do with P_GHE? How  does it connect to how a ground-source heat pump operates? 
 - "Wärme/Kälte durch Wärmepumpe": It would be better to have it as E = Q / COP, since they typically know the ideal heating loads and wanna figure out how much their electricity loads increase when operating an HP.
 - "Wärme durch Verbrennung": An equation telling them the total operating cost and carbon emissions based on their heating loads and energy carrier would be much more useful

In my opinion, this formula sheet needs some update, keeping in mind what would be useful for architects

**Comment by hillias** [comment]
Agreed. Please propose alternative equations with references/background material keeping in mind that the lecture slides have to be updated accordingly.

**Comment by christophwaibel** [comment]
> Agreed. Please propose alternative equations with references/background material keeping in mind that the lecture slides have to be updated accordingly.

ahh, we already have this issue for it: https://github.com/architecture-building-systems/hive/issues/103

**Comment by hillias** [comment]
@amraladdin @christophwaibel Regarding Calculation sheet, the latest release is usually on moodle, e.g.: https://moodle-app2.let.ethz.ch/course/view.php?id=11111

In the future, we will also want to provide a copy of the calculation sheet on our A/S consultation platform hosted by Chirag (https://moodle-app2.let.ethz.ch/course/view.php?id=11917)

And we also might want to provide an english version, e.g. for our MBS students (second priority, first focus on german version)

In case you want to revise/update/expand the sheet, feel free to make suggestions directly in the word file and keep it somewhere, were we can track updates/versions (github?). The sheet has been revised a few times in the last couple years and, despite the fact it considerably improved, it is not yet, where it should be. Ideas for further improvement:

- Critically review the equations and variables whether they make sense for early design stage calculations (HIVE and EK1-3 lectures). They should be as simple as possible yet correctly reflect dependencies between design choices and performance (I think an accuracy of 20-30% is acceptable).
- Nomenclature/Use of symbols should be consistent throughout the sheet (and HIVE).
- Add typical values for selected parameters (e.g. typical efficiencies, COP's, ... similar to the drop-down? menus planned for HIVE).
- Instead of equations, rule of thumbs might be added where appropriate.
- Add comments to each topic/equation which show origin, simplifications and/or references in case someone wants to get further information or look up, how the equation was derived. Most of the current equations are from the lecture slides, sometimes with missing references :-(, or from SIA 2024/380.

Ultimately, Arno will need to update his lecture slides to match the equation sheet and input/ouputs from HIVE. I'm assuming, he would want to do this in the summer break. I.e. by August he would need a final version of the equation sheet.
 

[EK1_Formelsammlung_HS19.docx](https://github.com/architecture-building-systems/hive/files/4134174/EK1_Formelsammlung_HS19.docx)

**Comment by ChrisZenhub** [comment]
discussing this issue on meeting next Tuesday (7th April) moerning 9:00 with @hillias @amraladdi and arno

**Comment by amraladdin** [comment]
Shouldn't this issue be concluded?

**Comment by christophwaibel** [comment]
> Shouldn't this issue be concluded?

it should, but it isn't ;( the checklist on the top needs to be completed. but that's mainly my job, so I'll remove @amraladdin and @hillias as assignees

**Comment by amraladdin** [comment]
Ok great :)

**Comment by ChrisZenhub** [comment]
Had a talk with Martin Mosteiro. Use his methods for multi-energy-systems sizing also in Hive as a first step!

Check out his mail that he sent to me on 2020-05-28

**Comment by ChrisZenhub** [comment]
for the MVP, we have:

- CHP
- Gas Boiler
- ASHP
- PV
- ST
